tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -226,6 +227,16 @@ internal static void InitializeNoNativeParts()
                 Log.Error(ex, ex.Message);
             }
 
+            try
+            {
+                Log.Debug("Initializing live debugger singleton instance.");
+                _ = LiveDebugger.Instance;
```

Comment:
It might be better to initialize this right after the Tracer is initialized, since exporter settings may be shared between the two

Reply:
Yes, you have a point here.
I've moved initialization right after tracer Tracer at commit c699b195331070fa1cccd6dd5f46e332266fe5a2
For now I don't want to re-use exporter settings, it will require a bit of refractory, but for sure I will do so, when will work on removing `ImmutableDebuggerSettings` as discussed in another comment with @andrewlock